File: app/src/main/java/id/ac/ui/cs/mobileprogramming/nabilafv/rememberyourlord/dao/EnTransDao.java
package id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.dao;

import androidx.lifecycle.LiveData;
import androidx.room.Dao;
import androidx.room.Query;

import java.util.List;

import id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.model.EnTrans;

@Dao
public interface EnTransDao {

    @Query("SELECT * FROM en_trans")
    LiveData<List<EnTrans>> fetchAllEnTrans();

    @Query("SELECT * FROM en_trans WHERE ID = :id")
    LiveData<EnTrans> getEnTrans(int id);
}

File: app/src/main/java/id/ac/ui/cs/mobileprogramming/nabilafv/rememberyourlord/dao/EnTransDao.kt
package id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.dao

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Query
import id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.model.EnTrans

@Dao
interface EnTransDao {
    @Query("SELECT * FROM en_trans")
    fun fetchAllEnTrans(): LiveData<List<EnTrans>>

    @Query("SELECT * FROM en_trans WHERE ID = :id")
    fun getEnTrans(id: Int): LiveData<EnTrans>
}